lms/templates/admin/application_instance/show.html.jinja2

```diff
@@ -12,12 +12,16 @@
         {% endif %}
     </span>
 {% endblock %}
-{% macro settings_checkbox(label, setting, sub_setting, field_name, default=False) %}
+{# Tri-state checkbox with enabled/disabled/use-default options. #}
+{% macro settings_checkbox(label, setting, sub_setting, field_name, default=None) %}
```

Comment:
We probably should keep `default` as `False`, maybe changing the name of the parameter? `default_label` or similar.

This parameter is no longer used to set a default value but it could be used to render the default label.

I don't think it's used at all right now.